tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Startup.cs

```diff
@@ -504,6 +504,10 @@ private void CreateConfig(IHostingEnvironment hostingEnvironment)
                                             {
                                                 jw.WriteValue(bool.Parse(subKey.Value));
                                             }
+                                            else if (subKey.Key == "displayInDashboard")

```

Comment:
This setting is for configuring which columns needs to be displayed in the dashboard pages